Detectors/FIT/common/calibration/include/FITCalibration/FITCalibrationApi.h

```diff
@@ -0,0 +1,96 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef O2_FITCALIBRATIONAPI_H
+#define O2_FITCALIBRATIONAPI_H
+
+#include <utility>
+
+#include "CCDB/CcdbApi.h"
+#include "TH1F.h"
+#include "TGraph.h"
+#include "Rtypes.h"
+
+
+namespace o2::calibration::fit
```

Comment:
Is this class supposed to proved calibration in the workflow? We assume that the DPL framework will provide calibrations in a same way as input messages. BTW, note that the CcdbApi does not support caching, while BasicCCDBManager does.

Reply:
So should calibration object be received from the previous DPL in the workflow? Isn't that more effiecent to read it only once and then update without touching ccdb?

Reply:
It is not functional yet, but the idea is that device subscribes to the CCDB objects in the same way as if does for the input data and the framework provides them, taking care about timestamp validity, caching etc.
What do you mean by updating ccdb object? The objects received from CCDB must be treated as const.
Data processing (to be reproducible) must use calib objects as they are provided by the CCDB.

Reply:
Well by updating ccdb object I mean that I have locally stored copy in `FITCalibrationApi ` instance which I'm updating right after `finalizeSlot` and perfoming calibration algorithm. Then I'm using this object to calibrate incoming input data. But it seems from your comments that is not the right approach...